feature/xhr/aggregate/index.js

```diff
@@ -98,7 +98,7 @@ function storeXhr(params, metrics, startTime, endTime, type) {
     spaAjaxEvents[interactionId] = spaAjaxEvents[interactionId] || []
     spaAjaxEvents[interactionId].push(event)
   } else {
-    ajaxEvents.push(event)
+    if (allAjaxIsEnabled()) ajaxEvents.push(event)
```

Comment:
I would recommend moving the `allAjaxIsEnabled` call to line 78. That way events will not be created at all.